app/user/rest.py

```diff
@@ -454,6 +457,26 @@ def send_support_email(user_id):
     return jsonify({"status_code": status_code}), 204
 
 
+@user_blueprint.route('/<uuid:user_id>/contact-request', methods=['POST'])
+def send_contact_request(user_id):
+
+    try:
+        contact = ContactRequest(**request.json)
+        user = get_user_by_email(contact.email_address)
+        if not any([not s.restricted for s in user.services]):
+            contact.tags = ['z_skip_opsgenie', 'z_skip_urgent_escalation']
+
+        status_code = Freshdesk(contact).send_ticket()
+        if 200 <= status_code <= 299:
+            status_code = ZenDeskSellClient(contact).send_contact_request()
+
+        return jsonify({"status_code": status_code})
+    except (NoResultFound, TypeError) as e:
+        print(e)
```

Comment:
`get_user_by_email` may return no result if you don't have an account and that's fine